shared/modules/content-security-policy.ts

```diff
@@ -0,0 +1,65 @@
+// ASCII whitespace is U+0009 TAB, U+000A LF, U+000C FF, U+000D CR, or U+0020 SPACE.
+// See <https://infra.spec.whatwg.org/#ascii-whitespace>.
+const ASCII_WHITESPACE_CHARS = '\t\n\f\r ';
+const ASCII_WHITESPACE = RegExp(`[${ASCII_WHITESPACE_CHARS}]+`);
+const ASCII_WHITESPACE_AT_START = RegExp(`^[${ASCII_WHITESPACE_CHARS}]+`);
+const ASCII_WHITESPACE_AT_END = RegExp(`[${ASCII_WHITESPACE_CHARS}]+$`);
```

Comment:
minor nit, feel free to ignore: for consistency with other parts of our codebase these should probably use `new RegExp` (not just `RegExp`). (I'm surprised we don't have an eslint rule that automatically fixes this)